### PR TITLE
chore: documentation and CI updates

### PR DESCRIPTION
**Pull request by Fdawgs**

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by Fdawgs** [comment]
@mcollina, mind if i open similar requests on other pino repos?
Some don't have CI in place (or are on travis still), so wanted to fix that!

**Comment by mcollina** [comment]
go for it!

File: package.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
     "pino-pretty": "./bin.js"
   },
   "scripts": {
-    "ci": "standard && tap 'test/**/*.test.js' --coverage-report=lcovonly",
+    "ci": "standard && tap \"test/**/*.test.js\" --coverage-report=lcovonly",
     "lint": "standard | snazzy",
-    "test": "tap --100 'test/**/*.test.js'"
+    "test": "tap --100 \"test/**/*.test.js\""
```
Comment (jsumners) [inline package.json:12]: These changes should not have been made. The `"` will introduce shell expansion of `**/*.test.js`. The `'` prevents shell expansion and allows the `glob` package to do the expansion.

@Fdawgs please do not blanket change quotes.

Comment (Fdawgs) [reply package.json:12]: Ah, sorry @jsumners. Is there a better way of doing this?

The Fastify repos all use a similar method to ensure the tests run on Windows.

Comment (jsumners) [reply package.json:12]: They should all be using single quotes.

Comment (Fdawgs) [reply package.json:12]: After further investigation, I don't think it matters if single or double quotes are used when it's just globbing (phew, no mass opening of PRs on all of Fastify's repos):

```bash
$ cd test

$ echo *
decorate.test.js hook.test.js integration.test.js plugin.test-d.ts

$ echo "*"
*

$ echo '*'
*
```

**Pull request merged**